cloudhub-2/modules/ROOT/pages/ch2-patch-updates.adoc

```diff
@@ -21,6 +21,147 @@ When an update is available, CloudHub 2.0 includes it in Production environments
 
 xref:cloudhub::vpn-maintenance.adoc[Anypoint VPNs] used in CloudHub 2.0 are upgraded continuously. Whenever a new update is available, the VPN is patched within the service-level agreement. If the Anypoint VPN is not setup as high availability or if the non-high-availability VPN is not set up with two tunnels, downtime is expected.
 
+== Self-Triggered Infrastructure Upgrades
```

Comment:
@aishwarya-murali  should we be documenting that self upgrade feature is behind the feature flag.

Reply:
decided to not hide behind feature flag